internal/services/netapp/validate/volume_group_sap_hana_volumes_validation.go

```diff
@@ -67,8 +67,8 @@ func PossibleValuesForProtocolTypeVolumeGroupSapHana() []string {
 type SecurityStyle string
 
 const (
-	SecurityStyleUnix SecurityStyle = "Unix"
-	SecurityStyleNtfs SecurityStyle = "Ntfs"
+	SecurityStyleUnix SecurityStyle = "unix"
```

Comment:
could we remove these and use the enum mentioned above instead?

Reply:
Updated to use enum values in go azure sdk.